### PR TITLE
Add example project

### PR DESCRIPTION
**Pull request by lordkev**
Also updated the README as it was using an out of date method definition for caching.

@dmauro

### CODE REVIEW

File: MattressExample/MattressExample/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+import UIKit
+import Mattress
+
+class ViewController: UIViewController {
+
+    @IBOutlet var webView: UIWebView!
+    let urlToCache = NSURL(string: "https://www.google.com")
+
+    override func viewDidLoad() {
```
Comment (dmauro) [inline MattressExample/MattressExample/ViewController.swift]: Get rid of boilerplate

**Comment by dmauro** [comment]
Mattress and CommonCrypto aren't already built, and there's no Cartfile or Podfile.

**Comment by dmauro** [comment]
Otherwise, looks good to me. Maybe some instructions somewhere explaining how to demo it:
1) Launch app.
2) Press load page, wait for "Finished caching" log
3) Turn off internet access
4) Tap on load page and web page will load.

**Comment by lordkev** [comment]
If you open it from the workspace it should build the framework target as a dependency.

**Comment by dmauro** [comment]
Oh got it. FWIW my instinct was to go straight to the example folder and open that project file.

**Comment by lordkev** [comment]
Added instructions including opening the workspace. :)

**Pull request merged**